services/database-analyzer.js

```diff
@@ -127,6 +127,20 @@ function DatabaseAnalyzer(databaseConnection, config) {
     tableForeignKeysAnalyzer = new TableForeignKeysAnalyzer(databaseConnection, config);
     columnTypeGetter = new ColumnTypeGetter(databaseConnection, config.dbSchema || 'public');
 
+    if (config.dbSchema) {
+      const schemaExists = await queryInterface.sequelize
+        .query(
+          `SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = '${config.dbSchema}';`,
+          { type: queryInterface.sequelize.QueryTypes.SELECT },
+        )
+        .then(result => !!result.length);
```

Comment:
Maybe `Boolean(result.length)` is more expressive (`!!result.length` could be considered as a (micro) hack). Not sure about your best practices though, so as you want.